RFS/src/main/java/com/rfs/worker/WorkerStep.java

```diff
@@ -0,0 +1,6 @@
+package com.rfs.worker;
+
+public interface WorkerStep {
```

Comment:
Why do you need an interface, aren't these something like Callable<WorkerStep>?

Reply:
It seemed like a good idea at the time, though that could just be because I'm a Java noob.  If there's a better way to do it I'm all ears.